src/Loop.php

```diff
@@ -124,7 +120,8 @@ public static function get()
      */
     public static function stop()
     {
-        self::get()->stop();
+        $driver = self::$driver ?: self::get();
```

Comment:
Haha, I did the same mistake. `self::createDriver()` just creates it and returns it, but doesn't assign it.